app/views/incomeSources/manage/ManageIncomeSources.scala.html

```diff
@@ -0,0 +1,172 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import _root_.implicits.ImplicitDateFormatterImpl
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.viewmodels.ViewIncomeSourcesViewModel
+
+
+@this(
+        mainTemplate: unifiedLayout,
+        implicitDateFormatter: ImplicitDateFormatterImpl,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        hr: hr,
+        govukTable: GovukTable
+)
+
+@(
+        sources: ViewIncomeSourcesViewModel,
+        isAgent: Boolean,
+        backUrl: String
+)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@import implicitDateFormatter.longDate
+
+@getMessage(key: String, args: String*) = @{
+    messages(s"view-income-sources.$key", args: _*)
+}
+
+@viewSelfEmploymentDetailsURL = @{
+    if(isAgent){
+        controllers.incomeSources.manage.routes.ManageSelfEmploymentSourceController.showAgent().url
+    } else {
+        controllers.incomeSources.manage.routes.ManageSelfEmploymentSourceController.show().url
+    }
+}
+
+@ukProperty = @{ sources.viewUkProperty }
```

Comment:
* its seem we are using copy/paste approach from the similar stories;
* I don't think we really need these variable definitions at all;

Reply:
Why would I not need them?

Reply:
its really up to you, but use of these one line variables seems to be redundant